webapp/src/components/admin_console_settings/rtcd_service_url/index.tsx

```diff
@@ -72,7 +81,7 @@ const RTCDServiceUrl = (props: CustomComponentProps) => {
                 <div className={'control-label ' + leftCol}>
                     <LabelRow>
                         <span>{props.label}</span>
-                        <UpgradePill>{'Enterprise feature'}</UpgradePill>
```

Comment:
Changed this so that it fits nicely even when setting name is longer (such as in the recordings case). Please let me know if it's an issue.

![image](https://user-images.githubusercontent.com/1832946/204887165-45178822-ebe7-4422-85a3-8b1f173534bd.png)

![image](https://user-images.githubusercontent.com/1832946/204887254-0d50587d-9e3c-47c3-8f78-f93e1700ceed.png)

Reply:
Nice call! I agree, removing the "Feature" makes this fit and look better